src/test/java/lotto/domain/LottoNumberTest.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class LottoNumberTest {
+
+    @Test
+    public void 로또숫자_범위가_1이상_45이하_테스트_45초과일_때(){
+        Assertions.assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(()->new LottoNumber(50));
+    }
+
+    @Test
+    public void 로또숫자_범위가_1이상_45이하_테스트_1미만일_때(){
```

Comment:
테스트명에는 결과도 함께 추가되면 좋을 것 같아요!!

```suggestion
    public void 로또숫자_범위가_1이상_45이하_테스트_1미만일_때는_에러가_발생(){
```